MIGRATION_TO_CACTI.md

```diff
@@ -0,0 +1,73 @@
+<!--
+ Copyright IBM Corp. All Rights Reserved.
+
+ SPDX-License-Identifier: CC-BY-4.0
+ -->
+# Migration to Cacti
+
+If you are already familiar with the Weaver Getting Started tutorial, you can compare it with the corresponding tutorial for Weaver components within Cacti. The instructions are almost identical, with differences in the names of packages, images, and modules, either in configuration files or in public repositories.
+
+## Repository
+
+The entire source code of `github.com/hyperledger-labs/weaver-dlt-interoperability` (https://github.com/hyperledger-labs/weaver-dlt-interoperability) was migrated to the `weaver` folder within `github.com/hyperledger/cacti` (https://github.com/hyperledger/cacti/tree/main/weaver). As a developer/contributor, you can compare the two and note recent changes made to the latter.
+
+## Documentation
+
+Weaver philosphophy and usage was documented in https://labs.hyperledger.org/weaver-dlt-interoperability/docs/external/introduction. The philosophy and usage of Weaver components as part of the Cacti framework are now documented in https://hyperledger.github.io/cacti/weaver/introduction/.
+
+Users can compare the instructions for setting up test networks and testing protocols in https://labs.hyperledger.org/weaver-dlt-interoperability/docs/external/getting-started/guide with the newer Cacti version in https://hyperledger.github.io/cacti/weaver/getting-started/guide/. The instructions look very similar; only the references (URLs, package names, Docker image names, etc.) have changed.
+
+## References: Package and Image Names
+
+Below we list the published Weaver packages and images, and their corresponding published Cacti packages and images.
+
+### Packages
+
+| Type          | Module/Feature | Weaver                               | Cacti |
+|---------------|-------------------------------------|----------|-------------------------------------------------------------------------|
+| NPM (JavaScript & TypeScript) | Common Protobufs (JS/TS) | [@hyperledger-labs/weaver-protos-js](https://github.com/orgs/hyperledger-labs/packages/npm/package/weaver-protos-js) | [@hyperledger/cacti-weaver-protos-js](https://github.com/hyperledger/cacti/pkgs/npm/cacti-weaver-protos-js) |
+|  | Node SDK for Fabric | [@hyperledger-labs/weaver-fabric-interop-sdk](https://github.com/orgs/hyperledger-labs/packages/npm/package/weaver-fabric-interop-sdk) | [@hyperledger/cacti-weaver-sdk-fabric](https://github.com/hyperledger/cacti/pkgs/npm/cacti-weaver-sdk-fabric) |
+|  | Node SDK for Besu | [@hyperledger-labs/weaver-besu-interop-sdk](https://github.com/orgs/hyperledger-labs/packages/npm/package/weaver-besu-interop-sdk) | [@hyperledger/cacti-weaver-sdk-besu](https://github.com/hyperledger/cacti/pkgs/npm/cacti-weaver-sdk-besu) |
+| Go | Common Protobufs (Go) | `github.com/hyperledger-labs/weaver-dlt-interoperability/common/protos-go` | `github.com/hyperledger/cacti/weaver/common/protos-go` |
```

Comment:
Can you add `/v2` at the end of every `go` package? That is required because Cacti is v2.0.0-alpha.2, otherwise people might not be able to get the package without that.

Reply:
Fixed